src/pages/home/Home.tsx

```diff
@@ -89,6 +89,16 @@ export default function Home() {
                         Give feedback
                     </CategoryButton>
                 </Link>
+                <a
+                    href="https://revolt.social"
```

Comment:
Why not just pull the servers from the website?

Reply:
That's coming in the future, this is just a solution for the time being since we are having so much traffic recently.